DEVELOPERS.md

```diff
@@ -148,10 +153,26 @@ To debug code, and to see changes in real time, it is often useful to have a loc
     http://localhost:8082/
     ```
 
-For more information on Supabase Studio, see the [Supabase Studio readme](./studio/README.md).
```

Comment:
Let's not delete this, it leads to more in-depth docs on how to contribute to the studio.

Reply:
Okay will add those back then